Paintroid/src/androidTest/java/org/catrobat/paintroid/test/espresso/tools/LineToolIntegrationTest.java

```diff
@@ -391,6 +411,7 @@ public void testClickingUndoOnceOnConnectedLines() {
 				.checkPixelColor(Color.BLACK, BitmapLocationProvider.HALFWAY_RIGHT_MIDDLE);
 
 		onTopBarView().performClickCheckmark();
+		IdlingRegistry.getInstance().unregister(idlingResource);
```

Comment:
You already invoke .unregister() in the tearDown method, so I think there is no need to additionally .register() at the start of the test.